met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx

```diff
@@ -77,10 +77,32 @@ const CreateFolderForm = () => {
                 sx={{ marginBottom: '3em' }}
             >
                 <Grid item>
-                    <SecondaryButton onClick={() => setCreateFolderMode(true)}>Create Folder</SecondaryButton>
+                    <SecondaryButton
```

Comment:
please create a new button in commons with this style so we can reuse it later.

Reply:
sounds good